Include/httpClient/httpClient.h

```diff
@@ -3,12 +3,14 @@
 
 #pragma once
 #include "types.h"
+#include "task.h"
+#include "httpProvider.h"
```

Comment:
I think Cullen was suggesting that this doesn't need to be included in the base header, only people implementing a provider need it